### PR TITLE
hotplug_mem_simple: Add new case to test hotplug memory

### PR DESCRIPTION
**Pull request by huangyum**
Signed-off-by: Yumei Huang <yuhuang@redhat.com>
ID: 1903885

### CODE REVIEW

**Comment by huangyum** [comment]
@luckyh @xiangchunfu Pls help review.
 (1/1) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.1.x86_64.io-github-autotest-qemu.hotplug_mem_shm: PASS (125.14 s)

**Comment by xiangchunfu** [comment]
Tested machine type q35 and checked debug log. It meets our requirements. looks good to me. 
(1/1) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.1.x86_64.io-github-autotest-qemu.hotplug_mem_shm.q35: PASS

Acked by:Xiangchun Fu xfu@redhat.com

**Comment by yanan-fu** [comment]
It looks duplicate with `hotplug_memory.after.stress.hotplug.backend_file.no_policy.one.default` but with some different configuration, such as slot, maxmen, mem-path and so on ?

**Comment by yanan-fu** [comment]
@luckyh  @xiangchunfu  As my understanding, If the existing case can match kata test scenario with update cfg, we prefer to update cfg in test loop than add new case, isn't it ?

File: qemu/tests/cfg/hotplug_mem_shm.cfg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+- hotplug_mem_shm:
+    type = hotplug_mem_shm
+    mem_fixed = 2048
```
Comment (luckyh) [inline qemu/tests/cfg/hotplug_mem_shm.cfg:3]: Suggest `mem = 2048`

File: qemu/tests/cfg/hotplug_mem_shm.cfg [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    slots_mem = 10
```
Comment (luckyh) [inline qemu/tests/cfg/hotplug_mem_shm.cfg:4]: Suggest `slots_mem = 1`

File: qemu/tests/cfg/hotplug_mem_shm.cfg [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    maxmem_mem = 64855M
```
Comment (luckyh) [inline qemu/tests/cfg/hotplug_mem_shm.cfg:5]: Suggest `maxmem_mem = 4096M`

File: qemu/tests/cfg/hotplug_mem_shm.cfg [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    guest_numa_nodes = "node0"
+    mem_devs = "mem0"
+    use_mem_mem0 = "no"
+    size_mem_mem0 = 2048M
```
Comment (luckyh) [inline qemu/tests/cfg/hotplug_mem_shm.cfg:9]: Suggest `size_mem_mem0 = "${mem}M"`

**Comment by yanan-fu** [comment]
> It looks duplicate with `hotplug_memory.after.stress.hotplug.backend_file.no_policy.one.default` but with some different configuration, such as slot, maxmen, mem-path and so on ?

Eh~   Just noticed, `stress` is `only backend_ram.policy_default backend_file.policy_bind`, so no  `backend_file.no_policy` with `stress` for the existing case. A blocker for use the existing cases.

File: qemu/tests/cfg/hotplug_mem_shm.cfg [hunk 5]
```diff
@@ -0,0 +1,16 @@
+- hotplug_mem_shm:
```
Comment (luckyh) [inline qemu/tests/cfg/hotplug_mem_shm.cfg:1]: How about the name `hotplug_mem_simple`?

File: qemu/tests/cfg/hotplug_mem_shm.cfg [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    type = hotplug_mem_shm
```
Comment (luckyh) [inline qemu/tests/cfg/hotplug_mem_shm.cfg:2]: +1

Comment (huangyum) [reply qemu/tests/cfg/hotplug_mem_shm.cfg]: All updated as you suggested. Thanks.

**Comment by huangyum** [comment]
(1/1) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.1.x86_64.io-github-autotest-qemu.hotplug_mem_simple: PASS (125.38 s)

**Review by luckyh — APPROVED** [review]
LGTM

**Comment by yanan-fu** [comment]
With the restriction, no available existing cases, and this new one looks good to me.

Acked-by: Yanan Fu <yfu@redhat.com>

**Comment by luckyh** [comment]
Thanks all!

**Pull request merged**